### DISCUSSION TITLE
希望对图片的预览有一些限制设定

### DISCUSSION THREAD

**Discussion by JiaoYeTan** [libsgh/PanIndex#147] [Ideas]
比如超过多少gb的图片就不在提供预览功能
在阿里云盘中一些zip改后缀为jpg的文件也会进行预览,结果就是网速跑满了,而且也预览不出个啥来
![image](https://user-images.githubusercontent.com/36872913/137451153-3070c148-2cee-4f93-b608-dc585243fb95.png)

可以看到右下角已经跑了很高的网速,希望对这个能有一些优化

**Comment by libsgh** [comment]
好奇怪，文件本身是压缩包，改了个后缀就能预览出图片啦？这里应该预览失败才对

**Reply by JiaoYeTan** [reply]
这个应该是叫做图种吧,以前经常能见到

**Reply by libsgh** [reply]
为了分享压缩包也是够拼的了，新版可以指定后缀 如果影响较大 暂时先关闭这种后缀的预览
后面我加个预览文件大小的限制配置
